src/helpers/translations/translations.js

```diff
@@ -1,20 +1,21 @@
 export default {
   locale: "en",
   crypto: {
+    bluetext: "Always for sale",
     ethereum: "ethereum based",
     tokens: "tokens",
     subHeading:
-      "Let your digital assets make a valuable contribution to the world.",
+      "Link your social profile to your preferred public address and verifiably showcase your patronage on our platform.",
```

Comment:
```suggestion
      "Link your social profile to your address and showcase your patronage.",
```

I think this should be kept short.

I also wonder if we should say "showcase your giving/support/generosity" rather than "patronage"; even though it is expert mode.

Reply:
Yes giving is a better word. Definitely agree